marketplacetests/marketplace/regions/add_review.py

```diff
@@ -3,22 +3,24 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from marionette.by import By
-from gaiatest.apps.base import Base
 
+from marketplacetests.marketplace.app import Marketplace
+from marketplacetests.marketplace.regions.app_details import Details
 
-class AddReview(Base):
+class AddReview(Marketplace):
     """
     Page for adding reviews.
     """
 
+    _page_loaded_locator = (By.CSS_SELECTOR, 'form.add-review-form')
+
     _add_review_input_field_locator = (By.ID, 'review-body')
     _submit_review_button_locator = (By.CSS_SELECTOR, 'button[type="submit"]')
-    _review_box_locator = (By.CSS_SELECTOR, '.add-review-form')
     _rating_locator = (By.CSS_SELECTOR, ".ratingwidget.stars-0 > label[data-stars='%s']")
 
     def __init__(self, marionette):
-        Base.__init__(self, marionette)
-        self.wait_for_element_displayed(*self._review_box_locator)
+        Marketplace.__init__(self, marionette)
+        self.wait_for_page_loaded()
```

Comment:
It's initialized before even launching the app, e.g., [1], but looking at that now, I guess maybe we could include the `launch` in the `__init__`. I doubt there is anywhere that we need to just init `marketplace` and not `launch` it too. Although, if we include `launch` in the base `__init__` we're back to having code in the base `__init__` that is specific to the base object and not the children.

[1] https://github.com/mozilla/marketplace-tests-gaia/blob/master/marketplacetests/tests/test_marketplace_add_review.py#L19-L20